tests/acceptance/characters/control/test_rpc_control_blockchain.py

```diff
@@ -74,7 +74,7 @@ def test_bob_rpc_character_control_retrieve_with_tmap(
     wrong_tmap.prepare_for_publication(
         bob_encrypting_key=blockchain_bob.public_keys(DecryptingPower),
         alice_stamp=blockchain_alice.stamp)
-    wrong_tmap._blockchain_signature = os.urandom(64)    # ...because it only matters when Ursula looks at it.
+    wrong_tmap._blockchain_signature = b"this is not a signature, but we don't need one for this test....."  # ...because it only matters when Ursula looks at it. (65 bytes)
```

Comment:
See comment above about a constant for `65`

Reply:
Not sure I understand this one - are you suggesting going back to the random variant?

I like having pre-determined text as the placeholder here for debugging purposes.

Reply:
Yes, sorry, it was ambiguous. I think it is worth it to add an assertion to make sure the length of the text is actually the expected length of the signature.